dev/src/dev.clj

```diff
@@ -325,7 +325,7 @@
                   (doall res)
                   res)]
         ;; only throws an exception if the simple hydration makes a DB call
-        (when (pos-int? (call-count))
```

Comment:
I think this is a typo

Reply:
Oh sorry - this was intentional but I should have explained better. I don't want to merge this, but it's required to make tests pass because we're getting a false positive for an N+1 hydration query. I do have [this](https://github.com/metabase/metabase/pull/45848) PR up that should fix it.